templates/base.html

```diff
@@ -12,7 +12,8 @@
   <link rel="stylesheet" href="{% static 'css/bootstrap.simplex.min.css' %}">
   <link rel="stylesheet" href="{% static 'css/bootstrap-nav-wizard.css' %}">
 
-  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.10.1/css/all.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.10.1/css/v4-shims.css">
```

Comment:
In general I think this upgrade is perfectly fine. I'm curious though, what about v4 was blocking you from adding a new icon? Is it a problem that will affect other sites?

The only icon that I think didn't quite survive this transition is the `fa-globe` icon in the `Countries` nav-link in the top-level navbar. We should swap it out for [`fa-americas`](https://fontawesome.com/icons/globe-americas?style=solid) so that we can preserve that "earthy" feel.